### ISSUE TITLE
[Suggestion] crops

### ISSUE THREAD

**Issue by ChilledDayz** [xt9/simplyacceleration#1]
Just wondering if you would consider adding a machine to speed up crop growth.

**Comment by xt9** [comment]
Yes it's certainly something I'd like to add, but I'm swamped atm with other projects so it might be a while before I implement it.
